### ISSUE TITLE
Listing Request (BAL)

### ISSUE THREAD

**Issue by browserslab-md5** [forkdelta/tokenbase#389]
Contract address: 
0xA9d6a1c87422b211014f1009C541600b33344AFF

Issuer's official website: 
https://www.browserslab.com

Official contract address confirmation: 
Placed under the FAQ with the information about the contract

A link to a third-party review or discussion of the token:  
https://bitcointalk.org/index.php?topic=3070171
https://cryptocurrencytalk.com/topic/98512-annbal-browsers-lab-revolutionizing-software-automation/?tab=comments#comment-442983
https://foundico.com/ico/browsers-lab.html
https://www.coinhills.com/ico/view/browsers-lab/
https://icolistview.com/ico/details/browsers-lab/

Description: 
     Browsers Lab's ambitious goal is to make cloud-based testing easier. A platform to reduce the expense associated with managing, creating and executing tests. Leveraging our platform will reduce the overall effort with setting up different environments, and viewing the results. Deliver with even greater confidence as our dashboard and metrics provide insight into how the tests are performing. Any team can benefit from our platform, small or large. From an individual to enterprise scale. Browsers Lab has your back.
    Browsers Lab Token will help this platform grow and expand to fill the industries test automation needs. Regulated by smart contracts on an Ethereum based blockchain platform. Not only will the token help foster a much-needed platform, but will also create a new market. The Browsers Lab Token will facilitate individuals and companies by exchanging services in a secure manner.

**Comment by Kryptoniq** [comment]
There are only 5 token holders ATM.

**Comment by browserslab-md5** [comment]
Yes, we are getting ready for our presale. We will request it again after our presale and crowd-sale then thank you.

**Comment by Kryptoniq** [comment]
Thanks for your request, we have verified that the information provided is accurate. We'll be listing the token in the next few days, you'll receive an update when we do.
0xa9d6a1c87422b211014f1009c541600b33344aff, https://www.browserslab.com/

**Comment by browserslab-md5** [comment]
Thank you. We look forward to seeing it there.

**Comment by freeatnet** [comment]
Thank you for your request! Your token will be listed on April 10th, 2018. You will receive a notification of this issue closing once the listing is added to the platform (keep in mind there may be a delay of up to 30 minutes between the listing being added and all users seeing it).

Once listed, the token will appear for all users of ForkDelta (in the token list and in the ticker) and will be tradeable at a pre-defined URL featuring your token symbol. We will also announce the listing on our platforms: Reddit, Twitter, Telegram, and Discord.

Please let me know if you have any questions or concerns.

**Comment by freeatnet** [comment]
Thank you for your request! Browsers Lab Token has been listed: https://forkdelta.github.io/#!/trade/BAL-ETH

If you like our project, please consider [donating](https://forkdelta.github.io/about/#donate). Your donations keep the project running and are always appreciated.
